### ISSUE TITLE
[BUG] Failed %load commands hang notebook

### ISSUE THREAD

**Issue by bechbd** [aws/graph-notebook#13]
**Describe the bug**
When running a %load command that fails it continues to check which locks up the 

**To Reproduce**
1. Try use the %load command on an import that fails
2. After the "LOAD_FAILED" message appears the cell keeps refreshing
3. Attempt to run something in the other cell.  It is stuck on [*] waiting for the load cell
4. To fix you have to stop the kernel and reload the notebook

**Expected behavior**
After the LOAD_FAILED message is recieved I expect the retries to stop

**Comment by austinkline** [comment]
All we need is to the below line to check for any terminal status values for our bulk loader:

https://github.com/aws/graph-notebook/blob/main/src/graph_notebook/magics/graph_magic.py#L662

```
if interval_check_response["payload"]["overallStatus"]["status"] == 'LOAD_COMPLETED':
```

**Comment by austinkline** [comment]
This is done, addressed in #37
